src/Microsoft.Azure.WebJobs.Extensions.Kafka/Serialization/LocalSchemaRegistry.cs

```diff
@@ -43,26 +43,37 @@ public Task<List<string>> GetAllSubjectsAsync()
             throw new System.NotImplementedException();
         }
 
-        public Task<Schema> GetLatestSchemaAsync(string subject)
+        public Task<RegisteredSchema> GetLatestSchemaAsync(string subject)
         {
             throw new System.NotImplementedException();
         }
 
-        public Task<string> GetSchemaAsync(int id)
+        public Task<RegisteredSchema> GetRegisteredSchemaAsync(string subject, int version)
         {
-            return Task.FromResult(this.schema);
+            throw new System.NotImplementedException();
         }
 
         public Task<string> GetSchemaAsync(string subject, int version)
         {
             return Task.FromResult(this.schema);
         }
 
+        public Task<Schema> GetSchemaAsync(int id, string format = null)
+        {
+            var schema = new Schema(this.schema, SchemaType.Avro);
+            return Task.FromResult(schema);
+        }
+
         public Task<int> GetSchemaIdAsync(string subject, string schema)
```

Comment:
Is this method still part of the interface? If not, you can remove it.

Reply:
Yes. It is also Obsolete.